docs/admin/users/idp-sync.md

```diff
@@ -304,77 +339,141 @@ Visit the Coder UI to confirm these changes:
 
 ## Organization Sync (Premium)
 
-> Note: In a future Coder release, this can be managed via the Coder UI instead
-> of server flags.
-
 If your OpenID Connect provider supports groups/role claims, you can configure
 Coder to synchronize claims in your auth provider to organizations within Coder.
 
-First, confirm that your OIDC provider is sending clainms by logging in with
-OIDC and visiting the following URL with an `Owner` account:
+Viewing and editing the organization settings requires deployment admin
+permissions (UserAdmin or Owner).
 
-```text
-https://[coder.example.com]/api/v2/debug/[your-username]/debug-link
-```
+Organization sync works across all organizations. On user login, the sync will
+add and remove the user from organizations based on their IdP claims. After the
+sync, the user's state should match that of the IdP.
 
-You should see a field in either `id_token_claims`, `user_info_claims` or both
-followed by a list of the user's OIDC groups in the response. This is the
-[claim](https://openid.net/specs/openid-connect-core-1_0.html#Claims) sent by
-the OIDC provider. See
-[Troubleshooting](#troubleshooting-grouproleorganization-sync) to debug this.
+You can initiate an organization sync through the CLI or through the Coder
+dashboard:
 
-> Depending on the OIDC provider, this claim may be named differently. Common
-> ones include `groups`, `memberOf`, and `roles`.
+<div class="tabs">
 
-Next configure the Coder server to read groups from the claim name with the OIDC
-organization field server flag:
+### CLI
```

Comment:
Can we default to the `Dashboard` view? The cli is going to be more error prone and likely used after they understand the feature (via dashboard mucking around)